app/src/main/java/org/oppia/app/player/state/audio/AudioFragment.kt

```diff
@@ -0,0 +1,28 @@
+package org.oppia.app.player.state.audio
```

Comment:
Maybe model this as a top-level player subcomponent in case we ever want to support the audio bar for non-state cards in the future.

Reply:
Done. New structure: inside `player` we have `exploration`, `state` and `audio`